apps/admin-gui/src/app/admin/pages/admin-page/admin-services/service-detail-page/service-destinations/service-destinations.component.ts

```diff
@@ -18,34 +17,32 @@ import { RemoveDestinationDialogComponent } from '../../../../../../shared/compo
 })
 export class ServiceDestinationsComponent implements OnInit {
 
-  constructor(private route: ActivatedRoute,
-              private serviceManager: ServicesManagerService,
+  constructor(private serviceManager: ServicesManagerService,
               private notificator: NotificatorService,
               private translate: TranslateService,
               private dialog: MatDialog,
-              public authResolver: GuiAuthResolver
+              public authResolver: GuiAuthResolver,
+              private entityStorageService: EntityStorageService
               ) { }
 
   loading = false;
   filterValue = '';
 
-  serviceId: number;
+  service: Service;
   destinations: RichDestination[] = [];
   selection = new SelectionModel<RichDestination>(true, []);
 
   tableId = TABLE_FACILITY_SERVICES_DESTINATION_LIST;
 
   ngOnInit(): void {
     this.loading = true;
-    this.route.parent.params.subscribe(params => {
-      this.serviceId = params['serviceId'];
-      this.refreshTable();
-    });
+    this.service = this.entityStorageService.getEntity();
```

Comment:
In this component and page also remove on the button and filter field from the *ngIf the loading please